packages/block-library/src/audio/deprecated.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+export default [
+	{
+		save( { attributes } ) {
```

Comment:
I believe this deprecated version is missing attributes and support keys. (These should be the same as the current version)

Reply:
Good point; that will make the deprecation code more robust against future changes.